client/deposits/rename-notice/index.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import React, { useState, useEffect } from 'react';
+import { TourKit } from '@woocommerce/components';
+import { useDispatch } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export const PayoutsRenameNotice = () => {
+	const isPayoutsRenameNoticeDismissed =
+		wcpaySettings.isPayoutsRenameNoticeDismissed;
+	const { updateOptions } = useDispatch( 'wc/admin/options' );
+	const [ showTour, setShowTour ] = useState( false );
+
+	const onClose = () => {
+		updateOptions( {
+			wcpay_payouts_rename_notice_dismissed: true,
+		} );
+		setShowTour( false );
+		wcpaySettings.isPayoutsRenameNoticeDismissed = true;
+	};
+
+	useEffect( () => {
+		if ( ! isPayoutsRenameNoticeDismissed ) {
+			setShowTour( true );
+		}
+	}, [ isPayoutsRenameNoticeDismissed ] );
```

Comment:
I believe this does not have any effect because the value of `isPayoutsRenameNoticeDismissed` never changes.

What do you think @Jinksi ?

I will merge this PR but looks like we can use a follow up PR to improve here. Also, we might need more unit test for this component.